### DISCUSSION TITLE
Future support for vacums

### DISCUSSION THREAD

**Discussion by SebastianKristo** [Luligu/matterbridge#90] [General]
First of all, thanks for putting in the work for this quite frankly amazing project. I was just wondering if there were any plans to add support for vacuums?

**Comment by Luligu** [comment]
Hi,
thanks for your interest and your appreciation.

All device types will be supported of course but actually the issue is that the controllers don't support them. 
Almost all are still on Matter 1.1 (not entirely supported sometimes).

We are all still waiting for a general support of Matter 1.2 and 1.3.

My best

**Reply by Casuallynoted** [reply]
That's great to know! Do you know if the current iOS 18 beta has homekit support for Matter 1.2/1.3?

**Reply by Luligu** [reply]
I don't know but I heard that Apple wants to wait for Matter 1.4.
Let's see soon...
Even Matter 1.2 would be a dream by the way...
In the hap protocol they don't have the pressure!
